infoblox/ipam/README.md

```diff
@@ -0,0 +1,124 @@
+|------------|-----------|------------|---------------|-------------|
+| name | yes | string | empty | The host FQDN to which the IP is being assigned. Must contain the zone name (e.g. example.com) that the infoblox service account supports.  |
+| ipv4addrs | no(yes) | array of hashes | empty | Must declare an ipv4addrs or an ipv6addrs. This field is an array of hashes declaring the address to use. This string can be used to get the next available IP: [{ "ipv4addr":"func:nextavailableip:10.1.124.0/24" }]. To get a specific IP use this: [{ "ipv4addr":"10.1.124.53" }] |
+| ipv6addrs | no | array of hashes | empty | Must declare at least an ipv4addrs or an ipv6addrs. This field is constructed the same as ipv4addrs but with "ipv6addrs" in it. |
+| aliases | no | array of strings | empty | This is a list of aliases for the host. The aliases must be in FQDN format. This value can be in unicode format. |
+| allow_telnet | no | boolean | false | This field controls whether the credential is used for both the Telnet and SSH credentials. If set to False, the credential is used only for SSH. 
+| comment | no | string | empty | Comment for the record; maximum 256 characters. |
+| configure_for_dns | no | boolean | true | When configure_for_dns is false, the host does not have parent zone information. |
+| device_description | no | string | empty | The description of the device.
+| device_location | no | string | empty | The location of the device. |
+| device_type | no | string | empty | The type of the device. |
+| device_vendor | no | string | empty | The vendor of the device. |
+| disable | no | boolean | false | Determines if the record is disabled or not. False means that the record is enabled. |
+| disable_discovery | no | false | Determines if the discovery for the record is disabled or not. False means that the discovery is enabled. |
+| dns_aliases | no | array of strings | empty | The list of aliases for the host in punycode format. The name is a default dns name. |
+| use_ttl | no | boolean | false | Use flag for ttl parameter. If not used, default ttl is used for records. |
+| ttl | no | number | empty | The Time To Live (TTL) value for record. A 32-bit unsigned integer that represents the duration, in seconds, for which the record is valid (cached). Zero indicates that the record should not be cached. |
+
+#### Supported Actions
+
+| Action | API Implementation | Support Level |
+|--------------|:----:|:-------------:|
+| create | https://ipam.illinois.edu/wapidoc/objects/record.host.html | supported |
+| destroy | https://ipam.illinois.edu/wapidoc/objects/record.host.html | supported |
+| show | https://ipam.illinois.edu/wapidoc/objects/record.host.html | supported |
+| list_by_name | https://ipam.illinois.edu/wapidoc/objects/record.host.html#name | supported |
+| search | https://ipam.illinois.edu/wapidoc/objects/record.host.html#fields-list | supported |
+
+#### Supported Outputs
+= "_ref" - Infoblox host:record reference
```

Comment:
i think you mean for these to be **-** , not **=**
it doesn't render correctly otherwise